pkg/api/types.go

```diff
@@ -2303,8 +2303,9 @@ type FinalizerName string
 
 // These are internal finalizer values to Kubernetes, must be qualified name unless defined here
 const (
-	FinalizerKubernetes FinalizerName = "kubernetes"
-	FinalizerOrphan     string        = "orphan"
+	FinalizerKubernetes    FinalizerName = "kubernetes"
+	FinalizerOrphan        string        = "orphan"
+	FinalizerSynchronousGC string        = "collectingGarbage"
```

Comment:
Not sure how this started but why aren't we using normal constant rules for finalizers?  I.e. CamelCase